pkgs/by-name/rt/rtl_wmbus/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation rec {
+  release_tag = "d2be82c";
+  pname = "rtl-wmbus";
+  version = "0-${release_tag}";
+
+  src = fetchFromGitHub {
+    owner = "xaelsouth";
+    repo = "rtl-wmbus";
+    tag = release_tag;
+    hash = "sha256-TbNOvEyG0vNG68UpKRwJYDcUiDWTujneSD0Vx/4eFrA=";
+
+    # need to fetch revision from git
+    leaveDotGit = true;
+    postFetch = ''
+      # avoid having the whole .git-folder
+      substituteInPlace $out/Makefile --replace-fail "COMMIT_HASH?=\$(shell git log --pretty=format:'%H' -n 1)" "COMMIT_HASH:=$(git -C $out log --pretty=format:'%H' -n 1)"
+
+      rm -rf $out/.git
+    '';
+  };
+
+  # avoid reading the version from git to avoid fetching all tags
+  makeFlags = [
+    "TAG=${release_tag}"
+    "BRANCH="
+    "CHANGES="
+  ];
+
+  # make install would use /usr/bin
+  installPhase = ''
+    mkdir -p $out/bin
+    cp build/rtl_wmbus $out/bin
+  '';
```

```suggestion
  installPhase = ''
    runHook preInstall
    mkdir -p $out/bin
    cp build/rtl_wmbus $out/bin
    runHook postInstall
  '';
```

https://nixos.org/manual/nixpkgs/unstable/#sec-stdenv-phases:
> When overriding a phase, for example installPhase, it is important to start with runHook preInstall and end it with runHook postInstall, otherwise preInstall and postInstall will not be run. Even if you don’t use them directly, it is good practice to do so anyways for downstream users who would want to add a postInstall by overriding your derivation.